xml/obs_best_practice_staging_workflow.xml

```diff
@@ -8,16 +8,22 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink" >
   <title>Staging</title>
-  <para></para>
+  <para>
+    At the moment, it is possible to create a Staging for any kind of project
+    unless the project is already one of the staging projects.
+  </para>
+  <para>
+    All the Staging workflow starts in the tab called 'Staging' that we can
```

Comment:
start<strike>s</strike> in the tab 'Staging' which can be found on the project's page.